cloud/azure/bin/lib/key_vault.sh

```diff
@@ -57,6 +57,21 @@ function key_vault::add_secret() {
     --value "${3}" > /dev/null
 }
 
+#######################################
+# Add a secret to the key vault
+# Arguments:
+#   1: The name of the key vault
+#   2: The name of the secret (used to identify it e.g. "postgres-password")
+#######################################
+function key_vault::add_secret_from_input() {
+  local SECRET
+  echo "Please enter the value for ${2}: "
+  read -s $SECRET && unset REPLY
+  key_vault::add_secret "${1}" "${2}" "${SECRET}" && \
+    echo "Stored secret value for ${2} in key vault ${1}"
```

Comment:
suggestion (blocking): we `set -e` in all of our scripts, so these commands can be run sequentially with the same behavior
```suggestion
  read -s $SECRET
  unset REPLY
  key_vault::add_secret "${1}" "${2}" "${SECRET}"  
  echo "Stored secret value for ${2} in key vault ${1}"
```